config.js

```diff
@@ -0,0 +1,15 @@
+const starters = [
+  {
+    name: "nextjs",
+    repoUrl: "https://github.com/stackbit-themes/nextjs-starter",
+  },
+  {
+    name: "ts-mui-nextjs",
```

Comment:
can you make it into "ts-nextjs" and if needed we'll add other names and aliases later? (same as we don't say "tailwind" in the default one)
(yeah, we may want to change the repo name as well, but that's not a blocker)

Reply:
btw the exact commands the app will show are in this file
https://github.com/stackbit/stackbit-app/commit/5e3c8507aa18099e10efa9e5863e21ed7437bb7c